TimerGUI.pyw

```diff
@@ -1,120 +1,97 @@
+#GUI and commands code
+import tkinter as tk, time, math
 
-    #Set window to appear in the middle
+def createWindow():
+    
+    window = tk.Tk()
+    window.title('Timer lasera')
+    
     windowWidth = 330
     windowHeight = 122
-    screenWidth = initialWindow.winfo_screenwidth()
-    screenHeight = initialWindow.winfo_screenheight()
+    screenWidth = window.winfo_screenwidth()
+    screenHeight = window.winfo_screenheight()
 
     x_pos = math.floor((screenWidth - windowWidth) / 2)
-    y_pos = math.floor(((screenHeight - windowHeight) / 2) * 0.80)
+    y_pos = math.floor(((screenHeight - windowHeight) / 2) * 0.80) #multiply by 0.8 to make the window appear more to the top of the screen
+
+    window.geometry(f"{windowWidth}x{windowHeight}+{x_pos}+{y_pos}")
+    return window
 
-    initialWindow.geometry(f"{windowWidth}x{windowHeight}+{x_pos}+{y_pos}")
+def createLabel(window, text):
+    label = tk.Label(
+        window,
+        text = text,
+        width = 22,
+        height = 2,
+        font = 'Times 18 bold',
+        padx = 10,
+        pady = 10,
+        justify ='center')
+    return label
+
+def createButton(window, buttonText, bgColor, buttonCommand):
+    button = tk.Button(
+        window,
+        text = buttonText,
+        bg = bgColor,
+        justify = 'center',
+        font = 'Times 18 bold',
+        command = buttonCommand)
+    return button
+
+#first window creator 
+def createInitialWindow():
+    initialWindow = createWindow()
 
     #NO button command
     def shortWindowHide():
         initialWindow.withdraw()
-        time.sleep(2)  #2 minuty = 120
+        time.sleep(shortHideButtonDelay)
         initialWindow.deiconify()
 
     #YES button command
     def timerStart():
         initialWindow.withdraw()
-        time.sleep(2) #1h = 3600
+        time.sleep(timerDelay)
         createRecurringWindow()
         initialWindow.destroy()
 
     #Labels and Buttons for first window
-    isLoggerOn = tk.Label(
-        initialWindow,
-        text = 'Czy włączono logger lasera?',
-        width = 22,
-        height = 2,
-        font='Times 18 bold',
-        padx=10,
-        pady=10,
-        justify='center'
-        )
+    isLoggerOn = createLabel(initialWindow, 'Czy włączono logger lasera?')
     isLoggerOn.pack()
 
-    timerStartButton = tk.Button(
-        initialWindow,
-        text='TAK +1h',
-        bg='light green',
-        justify='center',
-        font='Times 18 bold',
-        command= timerStart
-    )
-    timerStartButton.pack(side='left', expand=True,fill=tk.BOTH)
-
-    shortWindowHideButton = tk.Button(
-        initialWindow,
-        text='NIE +2 min',
-        bg='red',
-        font='Times 18 bold',
-        command = shortWindowHide
-    )
-    shortWindowHideButton.pack(side='right', expand= True,fill=tk.BOTH)
-    initialWindow.mainloop()
+    timerStartButton = createButton(initialWindow, f'TAK +{round((timerDelay/3600))}h', 'light green', timerStart)
```

Comment:
Try `"TAK +{time}h".format(time=round((timerDelay/3600)))` for better readability